packages/data-core/src/types/types.ts

```diff
@@ -0,0 +1,48 @@
+export interface Schema {
+  name: string // "Article"
+  displayAttribute: string
+  attributes: {
+    [field: string]: string | { type: string }
+  }
+  relationships?: {
```

Comment:
if `relationships` is optional, why not `attributes` too? Is it valid to have a schema with relationships but not direct attributes?

Reply:
i see no reason why it _shouldn't_ be allowed, will update